wrf-cmake.rb

```diff
@@ -0,0 +1,74 @@
+# WRF-CMake (https://github.com/WRF-CMake/WRF).
+# Copyright 2019 M. Riechert and D. Meyer. Licensed under the MIT License.
+
+class WrfCmake < Formula
+  desc "The Weather Research and Forecasting (WRF) model with CMake support"
+  homepage "https://github.com/WRF-CMake/WRF"
+  url "https://github.com/WRF-CMake/WRF.git",
+      :branch => "wrf-cmake",
+      :commit => "8b9edc944f0cbbfb22966850ae3724f617f0f705"
+  version "4.1.wrf-cmake"
+  sha256 "37814ee7bfe7077cba8bd0175258ef763380fce3e7d8aab2ea8360a902a11043"
+  head "https://github.com/WRF-CMake/WRF.git", :branch => "wrf-cmake"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" # for gfortran
+  depends_on "jasper"
+  depends_on "libpng"
+  depends_on "netcdf"
+  depends_on "open-mpi"
+
+  resource "WPS" do
+    url "https://github.com/WRF-CMake/WPS.git",
+        :branch => "wps-cmake",
+        :commit => "d918dc6da17b177aa1fe0082866274968f45a171"
+  end
+
+  # Note: If you get an internal compiler error when building WRF,
+  # then you're likely running out of memory. To reduce memory usage,
+  # set HOMEBREW_MAKE_JOBS=1.
```

Comment:
In core caveats are only allowed for unexpected or changed behavior as a result of packaging with homebrew, but, this information strikes me as useful enough and important enough that, while this is in a private tap, it would make sense to add this as a caveat.

Reply:
I pushed a commit making this change. We can revert it if you like.

Reply:
It's great, I like it.